pages/common/gpg-tui.md

```diff
@@ -0,0 +1,36 @@
+# gpg-tui
+
+> Terminal user interface for GNU Public Guard.
+> More information: <https://github.com/orhun/gpg-tui>.
+
+- Start `gpg-tui`:
+
+`gpg-tui`
+
+- Start `gpg-tui` enabling color and ASCII armored output:
```

Comment:
```suggestion
- Start `gpg-tui` with color and ASCII armored output:
```

Reply:
Not sure about this. I personally prefer `enabling`.